Tests/js/views/ez-dashboardblockbasedview.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>eZ Dashboard Block Based view tests</title>
+</head>
+<body>
+<div class="container"></div>
+
+<script type="text/x-handlebars-template" id="dashboardblockbasedview-ez-template" >
+    <h1 class="dashboard-title">Dashboard</h1>
```

Comment:
please always put the minimal needed template for the test to run and not a copy paste of the "normal" template, so here you can remove the `h1`